### PR TITLE
Fix warnings related to code documentation

### PR DESCRIPTION
**Pull request by erychagov**

### CODE REVIEW

**Comment by msftgits** [comment]
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

File: MobileCenter/MobileCenter/Internals/Storage/MSFileUtil.h [hunk 1]
```diff
@@ -15,8 +15,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * Returns the `NSFileManager` instance used for reading and writing
  * directories.
- *
- * @returns
+ * @return inctance of `NSFileManager`
```
Comment (jaelim-ms) [inline MobileCenter/MobileCenter/Internals/Storage/MSFileUtil.h]: typo, *instance*

File: MobileCenter/MobileCenter/Internals/Channel/MSChannelDelegate.h [hunk 2]
```diff
@@ -14,16 +14,15 @@
 
 /**
  * Callback method that will be called before each log will be send to the server.
- * @param Instance of MSChannel.
+ * @param channel of MSChannel.
```
Comment (clpolet) [inline MobileCenter/MobileCenter/Internals/Channel/MSChannelDelegate.h]: "channel Instance of MSChannel." Would be more meaningful in this case.

File: MobileCenter/MobileCenter/Internals/Channel/MSChannelDelegate.h [hunk 3]
```diff
@@ -14,16 +14,15 @@
...
  * @param log The log to be sent.
  */
 - (void)channel:(id <MSChannel>)channel willSendLog:(id <MSLog>)log;
 
 /**
  * Callback method that will be called in case the SDK was able to send a log.
- * @param Instance of MSChannel.
+ * @param channel of MSChannel.
```
Comment (clpolet) [inline MobileCenter/MobileCenter/Internals/Channel/MSChannelDelegate.h]: "channel Instance of MSChannel." Would be more meaningful in this case.

**Comment by erychagov** [comment]
@capfei
I corrected your remarks. Could you check?

**Pull request merged**